src/components/Dashboard/SortList/SortList.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  ButtonGroup,
+  Button,
+} from 'react-bootstrap';
+import storage from './storage';
+import './SortList.scss';
+import { ReactComponent as SortIcon } from './sort.svg';
+import { ReactComponent as SortIconAsc } from './sort-asc.svg';
+import { ReactComponent as SortIconDesc } from './sort-desc.svg';
+
+export default class SortList extends Component {
+  storageKey = 'null';
+
+  constructor(props) {
+    super(props);
+    const { storageKey: key } = this.props;
+    this.storageKey = key;
+
+    const stateDefaults = {
+      currentField: null,
+      currentDirection: 0,
+    };
+    const savedState = this.storageKey ? storage.get(this.storageKey) : {};
+    this.state = { ...stateDefaults, ...savedState };
```

Comment:
Using default variable assignment is waaaay more understandable.

Reply:
Fixed